components/developer/gtk-doc/Makefile

```diff
@@ -9,44 +9,50 @@
 #
 
 #
+# Copyright 2021 Gary Mills
 # Copyright 2016 Alexander Pyhalov
 #
 
+BUILD_BITS=		64
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		gtk-doc
-COMPONENT_VERSION=	1.25
-COMPONENT_REVISION=	1
+COMPONENT_MAJOR_VERSION=        1.33
+COMPONENT_MINOR_VERSION=        2
+COMPONENT_VERSION=      $(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
+# COMPONENT_REVISION=	0
 COMPONENT_PROJECT_URL=	http://www.gtk.org/gtk-doc/
 COMPONENT_SUMMARY=	GTK+ DocBook Documentation Generator
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:1ea46ed400e6501f975acaafea31479cea8f32f911dca4dff036f59e6464fd42
-COMPONENT_ARCHIVE_URL=	http://ftp.gnome.org/pub/GNOME/sources/gtk-doc/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+    sha256:cc1b709a20eb030a278a1f9842a362e00402b7f834ae1df4c1998a723152bf43
+COMPONENT_ARCHIVE_URL=	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_MAJOR_VERSION)/$(COMPONENT_ARCHIVE)
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+# Set python version used by this product
+PYTHON_VERSION=		3.7
 
-PATH=$(PATH.gnu)
+# The built-in tests assume the package has already been installed
+TEST_TARGET=		$(NO_TESTS)
 
-CONFIGURE_OPTIONS+=	--sysconfdir=/etc
-CONFIGURE_ENV+=		PYTHON=$(PYTHON)
-CONFIGURE_ENV+=		PERL=$(PERL)
+include $(WS_MAKE_RULES)/common.mk
 
-# common targets
-build:		$(BUILD_32)
+PATH=$(PATH.gnu)
 
-install:	$(INSTALL_32)
+# Build configure script
+COMPONENT_PREP_ACTION += (cd $(@D); autoreconf -fiv)
 
-test:		$(NO_TESTS)
+CONFIGURE_OPTIONS+=	--bindir=$(USRBINDIR)
+CONFIGURE_OPTIONS+=	--sbindir=$(USRSBINDIR)
+CONFIGURE_OPTIONS+=	--sysconfdir=$(ETCDIR)
+CONFIGURE_ENV+=		PYTHON=$(PYTHON)
+CONFIGURE_ENV+=		PERL=$(PERL)
 
```

Comment:
COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
COMPONENT_TEST_ARGS =   -k -i
COMPONENT_TEST_TRANSFORMS += \
        '-n ' \
        '-e "/result_count/d" ' \
        '-e "/TOTAL:/p" ' \
        '-e "/SKIP:/p" ' \
        '-e "/PASS:/p" ' \
        '-e "/FAIL:/p" ' \
        '-e "/ERROR:/p" '

Reply:
Adding this (from solaris-userland) make the gmake test semi successful: 3 tests pass while 11 fail